addons/account/models/account_tax.py

```diff
@@ -1188,7 +1188,8 @@ def grouping_key_generator(base_line, tax_values):
 
         amount_total = amount_untaxed + amount_tax
 
-        display_tax_base = len(global_tax_details['tax_details']) == 1 and tax_group_vals['base_amount'] != amount_untaxed
+        display_tax_base = len(global_tax_details['tax_details']) == 1 and tax_group_vals_list[0]['base_amount'] != amount_untaxed \
+            or len(global_tax_details['tax_details']) > 1
```

Comment:
LGTM programmatically, but could you please add parenthesis grouping the first condition ? If only for the sake of clarity :) 
(you had me searching for Python operators priorities :) )